lms/djangoapps/instructor/tests/test_proctoring.py

```diff
@@ -61,13 +61,12 @@ def test_no_tab_non_global_staff(self):
         self.assertFalse(self.proctoring_link in response.content)
         self.assertFalse('Allowance Section' in response.content)
 
-    @patch.dict(settings.FEATURES, {'ENABLE_PROCTORED_EXAMS': False})
     def test_no_tab_flag_unset(self):
```

Comment:
@chrisndodge Here is another place where the feature flag is no longer being checked.